Changes

```diff
@@ -2,6 +2,9 @@ Change history for libwww-perl
 
 {{$NEXT}}
 
+6.48      2020-09-15 21:58:57Z
```

Comment:
This line (and the blank line before it) should be removed; the version and timestamp and filled in from the `{{$NEXT}}` template.